src/main/java/yale/task/TaskList.java

```diff
@@ -0,0 +1,265 @@
+package yale.task;
+
+import java.time.DateTimeException;
+import java.util.ArrayList;
+
+/**
+ * Customised ArrayList class called TaskList
+ * to contain Task objects
+ */
+public class TaskList {
+    /**
+     * ArrayList that can contain
+     * elements of type Task
+     */
+    ArrayList<Task> list;
```

Comment:
Might be good to name this variable something like tasks - a plural variable name